x-pack/plugins/fleet/scripts/create_agents/create_agents.ts

```diff
@@ -337,7 +337,7 @@ export async function run() {
       Array(currentBatchSize)
         .fill(0)
         .map(async (__, i) => {
-          const agentPolicyId = 'script-create-agent-' + uuidv4();
+          const agentPolicyId = uuidv4();
```

Comment:
a bonus fix, this was making the IDs artificially long which was messing with the runtime query